include/swift/AST/DiagnosticsSema.def

```diff
@@ -3645,7 +3645,8 @@ ERROR(decl_from_hidden_module,none,
       "in an extension with public or '@usableFromInline' members|"
       "in an extension with conditional conformances|"
       "in an extension with public, package, or '@usableFromInline' members|"
-      "in an extension with conditional conformances}1; "
+      "in an extension with conditional conformances|"
+      "in an exported conformance}1; "
```

Comment:
This diagnostic text is not ideal but I'd rather keep it simple until this is merged back to 6.0 to avoid complications.